recipes/diligent-core/all/conanfile.py

```diff
@@ -169,6 +172,11 @@ def package_info(self):
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Graphics", "GraphicsTools", "interface"))
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Graphics", "HLSL2GLSLConverterLib", "interface"))
 
+        if self.settings.os == 'Windows':
+            self.cpp_info.defines.append("FILE2STRING_PATH=%s" % os.path.join(self.cpp_info.rootpath, "bin", "File2String.exe")
+        else:
+            self.cpp_info.defines.append("FILE2STRING_PATH=%s" % os.path.join(self.cpp_info.rootpath, "bin", "File2String")
+
```

Comment:
```suggestion
```
FILE2STRING_PATH seems to be a cmake variable, and not a C++ macro, cf https://github.com/DiligentGraphics/DiligentCore/search?q=FILE2STRING_PATH